documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -24,15 +24,20 @@ Magma Access Gateway using [snap](https://snapcraft.io/magma-access-gateway).
 - Ubuntu 20.04 LTS
   ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
 
+> **NOTE**: Unless specified otherwise, all actions in this guide are executed using `root` 
+> account.
+
 ## 1. Install
 
 > **NOTES:**<br>
 > 1. To see the list of currently supported configuration options, execute:
 > ```bash
 > magma-access-gateway.install --help
 > ```
-> 2. Unless specified otherwise, all actions in this guide are executed using `root` account.<br>
-> 3. Installation progress can be monitored at all times using `journalctl`.
+> 2. Installation progress can be monitored at all times using `journalctl`.
+> 3. Installation process requires **at least one** system restart at the end of the process. 
+> Depending on given installation options/initial system state, another restart may be required 
+> after network configuration. When it happens, reconnect to the server once it's back online.
```

Comment:
Can you place this where this is expected (probably after the `magma-access-gateway.install`

Reply:
If there is an action to be done, example to "Reconnect", then it shouldn't be a "note".

Reply:
The idea behind having it here was to give user a heads-up - as you mentioned, user shouldn't be surprised by the fact that system is going down.

Reply:
Well i'd keep most of the note here but after the install part I'd add your message `After the system reboots, reconnect to it and monitor the installation progress in journalctl`

Reply:
Done.